addons/html_editor/static/tests/link/remove.test.js

```diff
@@ -18,7 +18,8 @@ describe("range collapsed", () => {
             stepFunction: async (editor) => {
                 await unlink(editor);
             },
-            contentAfter: '<p>a<span class="fa fa-music" contenteditable="false"></span>bcd[]e</p>',
+            contentAfterEdit:
+                '<p>a<span class="fa fa-music" contenteditable="false">\u200B</span>bcd[]e</p>',
```

Comment:
new spec ? (\u200B)

Reply:
I posted a question in the editor group this morning cause it's related to a fix PR from AGE. Since when we insert an icon without a link involved, there is a \u200B in the span of the icon. DMO and I think the \u200B shouldn't be removed. Actually, when I tested in the playground with the old web editor with a test case, the \u200B wasn't removed. So the spec in the old tests is probably not correct

But we can double check with AGE tomorrow when he is here :smiley_cat: